### PR TITLE
New package: Minion.MMOUI version 3.0

### PR DESCRIPTION
**Pull request by ghost**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/140279)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>     --- End of inner exception stack trace ---

(Automated response - build 776.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 

(Automated response - build 788.)

**Comment by stephengillie** [comment]
This package installs and launches normally on a Windows 10 VM. 

@Barradel,

The package was previously submitted under `GoodGameMods.Minion` and that PackageIdentifier seems more approriate. What do you think?

**Comment by Barradel** [comment]
@stephengillie, you are right, I think it seems more appropriate.

File: manifests/m/Minion/MMOUI/3.0/Minion.MMOUI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created with komac v2.0.4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Minion.MMOUI
```
Comment (stephengillie) [inline manifests/m/Minion/MMOUI/3.0/Minion.MMOUI.installer.yaml:4]:
```suggestion
PackageIdentifier: GoodGameModsLLC.Minion
```

Please update file name and path to match this change.

File: manifests/m/Minion/MMOUI/3.0/Minion.MMOUI.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created with komac v2.0.4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Minion.MMOUI
```
Comment (stephengillie) [inline manifests/m/Minion/MMOUI/3.0/Minion.MMOUI.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: GoodGameModsLLC.Minion
```

Please update file name and path to match this change.

File: manifests/m/Minion/MMOUI/3.0/Minion.MMOUI.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with komac v2.0.4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Minion.MMOUI
```
Comment (stephengillie) [inline manifests/m/Minion/MMOUI/3.0/Minion.MMOUI.yaml:4]:
```suggestion
PackageIdentifier: GoodGameModsLLC.Minion
```

Please update file name and path to match this change.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Feel free add your own commit to update this. The folders can either be renamed, or new folders made and the files moved, but the name and path have to be updated to match.

**Review by ghost — COMMENTED** [review]
Think its ok now.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-02-21T03:10:35.7687394Z ##[error] One or more errors occurred. (Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\m\Minion\GoodGameMods\3.0', expected 'manifests\g\GoodGameModsLLC\Minion\3.0') Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\m\Minion\GoodGameMods\3.0', expected 'manifests\g\GoodGameModsLLC\Minion\3.0'

(Automated response - build 792.)

File: manifests/m/Minion/GoodGameMods/3.0/Minion.GoodGameMods.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Created with komac v2.0.4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: GoodGameModsLLC.Minion
```
Comment (stephengillie) [inline manifests/m/Minion/GoodGameMods/3.0/Minion.GoodGameMods.installer.yaml:4]:
```suggestion
PackageIdentifier: GoodGameMods.Minion
```

File: manifests/m/Minion/GoodGameMods/3.0/Minion.GoodGameMods.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Created with komac v2.0.4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: GoodGameModsLLC.Minion
```
Comment (stephengillie) [inline manifests/m/Minion/GoodGameMods/3.0/Minion.GoodGameMods.yaml:4]:
```suggestion
PackageIdentifier: GoodGameMods.Minion
```

File: manifests/m/Minion/GoodGameMods/3.0/Minion.GoodGameMods.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# Created with komac v2.0.4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: GoodGameModsLLC.Minion
```
Comment (stephengillie) [inline manifests/m/Minion/GoodGameMods/3.0/Minion.GoodGameMods.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: GoodGameMods.Minion
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
The PackageIdentifier has to match the file name and path. The suggestion I made was updated to include "LLC', which was in the previous version, but this didn't get into the name or path.

**Comment by Barradel** [comment]
> the URLs for the installer are correct, that would help us to validate and approve this PR.

The URL for the installer is correct and can be verified just visiting the site: https://minion.mmoui.com/?download

File: manifests/g/GoodGameMods/Minion/3.0/GoodGameMods.Minion.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,14 @@
+# Created with komac v2.0.4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: GoodGameMods.Minion
+PackageVersion: '3.0'
+PackageLocale: en-US
+Publisher: Good Game Mods LLC
+PublisherUrl: https://minion.mmoui.com/
+PackageName: Minion
+PackageUrl: https://cdn.mmoui.com/minion/v3/Minion3.0.5.exe
```
Comment (stephengillie) [inline manifests/g/GoodGameMods/Minion/3.0/GoodGameMods.Minion.locale.en-US.yaml:10]:
```suggestion
PublisherUrl: https://www.mmoui.com/
PackageName: Minion
PackageUrl: https://minion.mmoui.com/
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
I didn't catch this before - the `PackageUrl` should point at the product webpage or similar. The `PublisherUrl` could match or could be updated, it's less important.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**